web/src/pages/dashboard/index.page.jsx

```diff
@@ -1,57 +1,62 @@
-const Sidebar = () => {
-    
-   return (
-       <section className="bg-white dark:bg-gray-900">
+// Layout.js
 
+import { HiTemplate, HiOutlineShoppingBag, HiOutlineUsers } from "react-icons/hi";
+import Link from 'next/link';
 
-           <div className="flex flex-col items-center md:h-screen lg:py-0">
+const SidebarData = [
```

Comment:
Pls use proper variable naming, I prefer links
```suggestion
const links = [
```